README.md

```diff
@@ -103,8 +103,8 @@ Role Variables
           `libvirt_volume_default_device` are valid here. Default is
           `libvirt_volume_default_type`.
         - `capacity`: volume capacity, can be suffixed with k, M, G, T, P or E when type is `network` or MB,GB,TB, etc when type is `disk` (required when type is `disk` or `network`)
-        - `auth`: Authentication details should they be required. If auth is required, `name`, `type`, `token` will need to be supplied.
-        - `source`: Where the remote volume comes from when type is `network`. `protocol`, `name`, `hostname` and `port` should be supplied.
+        - `auth`: Authentication details should they be required. If auth is required, `username`, `type`, and `uuid` or `usage` will need to be supplied. `uuid` and `usage` should not be both supplied.
+        - `source`: Where the remote volume comes from when type is `network`. `protocol`, `name`, and `hostname` or `hosts_list` should be supplied. `hostname` and `hosts_list` should not be both supplied. `port` is optional.
```

Comment:
I don't generally like changing options once added, but it's good to match the underlying XML, and this is still a new feature.

Reply:
It might be nice if hostname and hosts_list were more similar. Maybe hostname and hostnames?

Reply:
If the network volume support wer not already merged, I would remove `hostname` parameter and have only `hosts_list`.

Reply:
Well this change already breaks backwards compatibility...

Reply:
No, if we keep `hostname` parameter untouched like I did. I added `hosts_list` for that purpose.

Reply:
Do you want me to remove `hostname`?

Reply:
Changing name to username and token to uuid breaks backwards compatibility. I would probably rather remove hostname than keep inconsistently named fields.

Reply:
@goetzk suggested to change them because it's not that long since the original feature was merged. So you want the original parameters names ?

Reply:
I don't mind changing them to make them more consistent with the XML format (as you have). I'm just pointing out that we're already breaking backwards compatibility for those two, so why not also for hostname?

To be clear, let's remove hostname and keep just hosts_list.

Reply:
I agree with that :+1: